pandas/_libs/src/datetime/pd_datetime.c

```diff
@@ -255,5 +257,6 @@ static struct PyModuleDef pandas_datetimemodule = {
 
 PyMODINIT_FUNC PyInit_pandas_datetime(void) {
   PyDateTime_IMPORT;
+  import_array();
```

Comment:
So this needs to be called any time you import `ndarrayobject.h` right? If so, I wonder if it wouldn't be cleaner to move the include here to the header. That way other module that need these datetime functions can simply include / link the pd_datetime capsule and not have to include ndarrayobject.h directly. May be less confusing

Reply:
Yes (if you use any API, but if you don't you only need `ndarraytypes.h`) but once per compilation unit only (just fixed that).
Places that use `pd_datetime.h` don't need it (we just need to be sure the init code ran).